Engine/Module/ModuleEditor.cpp

```diff
@@ -102,7 +104,7 @@ update_status ModuleEditor::PreUpdate()
 update_status ModuleEditor::Update()
 {
 	//ImGui::ShowStyleEditor();
-	//ImGui::ShowDemoWindow();
+	ImGui::ShowDemoWindow();
```

Comment:
now we have the demo window?

Reply:
Just for debugging, it would be disabled in last commit